main.cpp

```diff
@@ -127,6 +130,14 @@ void Cellphone::connectToInternet(bool connectedToWifi)
     }
 }
 
+void Cellphone::changeScreenBrightness(int targetValue)
+{
+    while (screenBrightness < targetValue) 
+    {
+        std::cout << "Increasing screen size to: " << ++screenBrightness << std::endl;
```

Comment:
It's a bad idea to increment inside of expressions like this
it's better to put the increment/decrement outside of this expression, and use the variable inside the expression in a read-only context.
- [ ] revise: don't increment/decrement inside of `std::cout` expressions